sql/2025/05/featured-image-block-usage.sql

```diff
@@ -0,0 +1,61 @@
+# HTTP Archive query to get % WordPress sites using a block theme.
+#
+# WPP Research, Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See query results here: https://github.com/GoogleChromeLabs/wpp-research/pull/184
+SELECT
+  client,
+  COUNT(DISTINCT page) AS with_block_theme,
+  total_wp_sites,
+  COUNT(DISTINCT url) / total_wp_sites AS pct_total,
+  # For reference, include number of sites greater than or equal to WP 5.9, since only then block theme support was launched.
+  wp_gte_59
+FROM
+  `httparchive.crawl.pages`,
+  UNNEST(technologies) AS technology,
+  UNNEST(technology.info) AS version
```

Comment:
It's important to be careful with `UNNEST`ing in queries, as it can easily cause duplicated rows. The `UNNEST(technologies)` is definitely fine here because in the `WHERE` clause you limit it to WordPress, but I'm not sure about `UNNEST(technology.info)`.

I would recommend to avoid `UNNEST`ing and instead use a helper function like this one: https://colab.research.google.com/drive/1TmbJhSBuM-gTHA_E_W96pSjraQkFYpyi#scrollTo=BFifI50QnBAs

That way you don't `UNNEST` in the query itself and avoid duplicated rows with certainty.